Makefile.in

```diff
@@ -446,6 +446,7 @@ endif
 	$(INSTALL_DATA) -t $(inst_libdir) $(top_builddir)/libdecnumber/libdecnumber.a
 	ln -f -s $(SHARED_REALNAME_LIB) $(inst_libdir)/$(SHARED_LINKERNAME_LIB)
 	ln -f -s $(SHARED_REALNAME_LIB) $(inst_libdir)/$(SHARED_SONAME_LIB)
+	$(INSTALL) -d $(inst_libdir)/pkgconfig
 	$(INSTALL_DATA) -D -t $(inst_libdir)/pkgconfig $(top_builddir)/libdfp.pc
```

Comment:
Huh, that seems weird. Is there some racy condition between the two install commands below? I guess we don't need the -D options on the pkgconfig files below if we add this.

Reply:
Indeed. It does look something else caused the issue.
Let me remove this patch, leave just the GCC 11 test one and try to fix the CI integration.